cmd/ipfs/main.go

```diff
@@ -39,6 +39,7 @@ var log = logging.Logger("cmd/ipfs")
 var (
 	errUnexpectedApiOutput = errors.New("api returned unexpected output")
 	errApiVersionMismatch  = errors.New("api version mismatch")
+	errRequestCanceled     = errors.New("request canceled")
```

Comment:
err defined. I will make it exportable once I've found the place that needs the ctx-cancel-wrapped.